README.md

```diff
@@ -1050,91 +1401,33 @@ ninja install
-
-## CMake Presets
-
-**Work in progress!**
-
-### `CMakePresets.json`
+To build the documentation either specify the flag `--build-doc` to `oomph_build.py` or build the documentation manually, following the usual pattern:
+```bash
+cd doc
+cmake -G Ninja -B build
+cmake --build build
+```   
+There's no need for an installation.
 
-We provide a generic `CMakePresets.json` file in the root directory of the project. To list the available presets, run
+Note that building the documentation takes a long time and simply duplicates the documentation already available on the `oomph-lib` webpage. It is therefore mainly of interest to developers or other volunteers who want to provide new (or fix broken) tutorials. Once the build process is complete you can navigate your local copy of the `oomph-lib` webpages starting from `doc/html/index.html`. 
 
+To uninstall the documentation do
 ```bash
-cmake --list-presets
+ hierher Puneet -- help!
 ```
 
-We recommend that you can write your own `CMakeUserPresets.json` file. You can inherit your presets from the presets we provide in `CMakePresets.json`. For details on how to do this refer to the [CMake documentation](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
-
-**Remark:** We recommend that you do not use the Ninja Multi-Config generator yet.
+To add new tutorials to the doc directory, follow the same steps used for adding new demo drivers.
 
-**FIXME:** Sort out the clean-up for the multi-config generator. The install manifest doesn't specify the debug config lib files. Hmm...
 
-### `CMakeUserPresets.json` example
-
-```json
-{
-  "version": 5,
-  "configurePresets": [
-    {
-      "name": "macos_arm64",
-      "inherits": "base",
-      "displayName": "macos_arm64",
-      "generator": "Ninja",
-      "binaryDir": "${sourceDir}/build",
-      "cacheVariables": {
-        "CMAKE_APPLE_SILICON_PROCESSOR": "arm64",
-        "CMAKE_BUILD_TYPE": "Release"
-      },
-      "warnings": {
-        "unusedCli": true
-      }
-    }
-  ],
-  "buildPresets": [
-    {
-      "name": "macos_arm64",
-      "configurePreset": "macos_arm64",
-    }
-  ],
-  "testPresets": [
-    {
-      "name": "macos_arm64",
-      "inherits": "test-base",
-      "configurePreset": "macos_arm64"
-    }
-  ]
-}
-```
 
 ## Cheat sheet: autotools vs. cmake
+Earlier releases of `oomph-lib` used the autotools to build/install the library. To ease the conversion to the new way of doing things, the table below provides a side-by-side comparison of key operations. Note that we also provide a script, `scripts/***py`, ***hierher Puneet*** that (attempts to) converts an autotools `Makefile.am` into a corresponding `CMakeLists.txt`. It may need a bit work/checking but it tends to do most of the work.
 
-Assume that `$oomph_home_dir` is the `oomph-lib` home directory:
+In all the examples shown below we assume that `$oomph_home_dir` is the `oomph-lib` home directory:
 <table>
```

Comment:
Hmm, this table is really hard to view in the browser; it's too wide to fit easily on the screen. Not to mention it's not easy for a user just reading the RAW markdown to navigate in their text editor.  If we keep each table entry as a heading then it can also be navigated to from the table of contents